aiida_vasp/workchains/immigrant.py

```diff
@@ -0,0 +1,122 @@
+"""
+Verify workchain.
+
+-----------------
+Indented to be used to verify a calculation, perform corrections in inputs files and
+restart depending on physical principles etc. E.g. issues that are outside the Calculators awereness,
```

Comment:
Calculators to CalcJob?

Reply:
Oh, when I started writing this workchain, I copied it from `verify.py`, so I have to update this docstring. Thanks for pointing out. So your comment applied to `verify.py`.